client/src/locale/he_IL.js

```diff
@@ -100,6 +100,8 @@ const HEBREW_TRANSLATION = {
 	noPlansSavedTitle: 'טרם שמרת תוכניות',
 	ourFacebook: 'Facebook',
 	ourTwitter: 'Twitter',
+	ourInstagram: 'Instagram',
```

Comment:
we need to add the same texts also in Arabic (found in the same file, at the bottom